run: exit status 255
Unhandled exception:
Exception: case1 failed
#0      test_merge_sort (file:///workspace/mochi/tests/algorithms/x/Dart/sorts/unknown_sort.dart:108:5)
#1      _main (file:///workspace/mochi/tests/algorithms/x/Dart/sorts/unknown_sort.dart:119:3)
#2      _start (file:///workspace/mochi/tests/algorithms/x/Dart/sorts/unknown_sort.dart:130:3)
#3      main (file:///workspace/mochi/tests/algorithms/x/Dart/sorts/unknown_sort.dart:140:16)
#4      _delayEntrypointInvocation.<anonymous closure> (dart:isolate-patch/isolate_patch.dart:314:19)
#5      _RawReceivePort._handleMessage (dart:isolate-patch/isolate_patch.dart:193:12)
